Sources/PackageCollections/Model/Collection.swift

```diff
@@ -98,6 +99,16 @@ extension PackageCollectionsModel {
     }
 }
 
+extension PackageCollectionsModel.CollectionSource {
+    /// Returns the source's absolute file system path, if its URL is of 'file' scheme.
+    var absolutePath: AbsolutePath? {
+        guard self.url.scheme?.lowercased() == "file" else {
+            return nil
+        }
+        return try? AbsolutePath(validating: self.url.path)
```

Comment:
self.url.absoluteString

Reply:
`url.absoluteString` includes `file://` and `AbsolutePath` must start with `/` so the initialize will fail if we use it